command_line/export_best.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+
+from libtbx.phil import parse
+from libtbx import Auto
+
+logger = logging.getLogger("dials.command_line.export_best")
+
+help_message = """
+This program is used to export the results of dials processing for the strategy
+program BEST.
+
+"""
+
+phil_scope = parse(
+    """
+
+  intensity = *auto profile sum scale
```

Comment:
I don't think this `intensity` parameter is used in the best export.

Reply:
It _could_ be though... could imagine using it with scaled data for partial data sets (e.g. in an i19 context)

Reply:
Yes, but we shouldn't have unused program parameters that _might_ hypothetically be used in the future. If we decide to add such functionality at some point in the future then we can re-add the parameters to `dials.export_best`.

This is the code that extracts the intensities and writes them to `best.hkl`, and it doesn't currently use any parameter to control the choice of intensities:
https://github.com/dials/dials/blob/142bd8994a92bb9e50a5c6e9f08801e2118ed265/util/best.py#L21-L44

Reply:
Removed